redfish_client/resource.py

```diff
@@ -149,3 +156,32 @@ def wait_for(self, stat, expected, blacklisted=None, poll_interval=3, timeout=15
     @property
     def raw(self):
         return self._content
+
+    def post(self, attribute=None, payload=None):
+        """
+        Perform a POST at the resource with the given payload.
+
+        Args:
+          attribute: Optional attribute to POST to
+          payload: The contents of the POST payload.
+        """
+        if attribute is None:
+            path = self._content.get("@odata.id")
+        else:
+            path = self._content[attribute].get("@odata.id")
```

Comment:
Doing this makes little sense. For example

    root.AccountService.post("Accounts", dict(some="payload"))

call can be replaced by the

    root.AccountService.Accounts.post(payload=dict(some=payload))

The attribute field would come in handy for example when we would like to perform some action, since the URL of the action is specified in the `target` field. But at leas for now, I would simplify this and just remove the `attribute` argument completely, since actions are already covered. We can always add it in if the need arises.

Reply:
👍